src/components/input/code/templates/inputTemplate.jsx

```diff
@@ -0,0 +1,46 @@
+var React = require('react');
+
+module.exports = function (props){
+
+	var classes = classNames('input-group', {
+		'input-group-error': this.props.error || false,
```

Comment:
Would it be better to use [`getDefaultProps`](https://facebook.github.io/react/docs/component-specs.html#getdefaultprops) for setting default values?